pdp/static/js/pdp_raster_map.js

```diff
@@ -53,25 +87,31 @@ function dasToUnitsSince(data) {
         m = reg.exec(s),
         units = m[1],
         dateString = m[3],
-        sDate;
-
+        sDate;   
+    
+    var calendar;
+    reg = /calendar \"(standard|365_day|360_day)\"/,
```

Comment:
Also, if calendar comes in empty (it *shouldn't*, but just in case), you should make sure that you default to 'standard'.